bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Please rewrite every T with exponents before the T, such as ^{i}T_{j}. Please write p in lower case and as bold character. This is the convention for vectors in robotics. If you want to maintain the world, use only w, and for segment use s.</p><p> </p><p>And specify it in the text as "segment's frame is denoted with an s, and world's frame is denoted with a w."</p><p> </p><p>- "We need pass trough the world's frame"</p><p>Use inverse exponent -1 to show how to inverse the matrix.</p><p> </p><p>Insert symbolic characters in the sentence : "The position in i in relation to j is given by the rototranslation matrix between i and j, multiplied by the position in j."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>